clients/src/main/java/org/apache/kafka/common/config/AbstractConfig.java

```diff
@@ -66,52 +66,51 @@ public AbstractConfig(ConfigDef definition, Map<?, ?> originals) {
         this(definition, originals, true);
     }
 
-    protected Object get(String key) {
+    protected <T> T get(String key) {
```

Comment:
I am not sure about this. It's a really dangerous method as it implicitly casts in an unsafe manner.